18F/identity-design-system
# Split  "Form Fields" documentation to individual components

## 🛠 Summary of changes

Splits the existing "Form Fields" internal documentation page to individual pages per component.

For review: All of the example code variations were ported verbatim from the previous `form-fields.md` page, so it's not necessary to review the content / markup of these pages.

**Why?**

- It maps more cleanly to the actual underlying components
- Consistency with how [USWDS components are documented](https://designsystem.digital.gov/components/overview/)
- It's more useful, since individual components will now link to their individual USWDS source documentation and override styles
- It's more scalable as we add more form field component variations
- It should help make visual regression testing results more predictable and easier to parse

## 📜 Testing Plan

1. Go to https://federalist-340d8801-aa16-4df5-ad22-b1e3e731098e.sites.pages.cloud.gov/preview/18f/identity-design-system/aduth-split-form-fields/
2. Observe new component pages in side navigation
3. Click each new component
4. Confirm that leading information (title, lead, links to upstream resources) appear as expected
5. Confirm that existing documentation for components is unchanged from what existed previously with "Form Fields" documentation
